### PR TITLE
Groupby col

### PR DESCRIPTION
**Pull request by ericgitonga**
Make the patrols relocs use the patrol_id as the groupby-col instead of the subject_id

### CODE REVIEW

File: ecoscope/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.0.1"
+__version__ = "1.1.0"
```
Comment (walljcg) [inline ecoscope/version.py:1]: I think let's bump to 1.0.2 instead as I see this as a bug fix rather than introduction of new functionality

**Review by walljcg — CHANGES_REQUESTED** [review]

File: ecoscope/version.py (hunk 1 above)
Comment (ericgitonga) [reply ecoscope/version.py:1]: This bump was for the previous additions to Ecoscope... I had made the PR to bump to 1.1.0 before I did this change of patrol_id. So for patrol_id, it ought to be bumped to 1.1.2...

Comment (ericgitonga) [reply ecoscope/version.py]: I have bumped the version to 1.1.1

Comment (walljcg) [reply ecoscope/version.py]: Thanks

**Review by walljcg — APPROVED** [review]

**Pull request merged**